geos.c

```diff
@@ -189,11 +189,19 @@ getRelay(zval* val, zend_class_entry* ce) {
 
     if ( proxy->std.ce != ce ) {
         php_error_docref(NULL TSRMLS_CC, E_ERROR,
+#if PHP_VERSION_ID >= 70000
```

Comment:
Shall .travis.yml be updated to include the new PHP version ?

Reply:
Yes

Reply:
Please do so in the PR. We want to test both old and new php version